README.md

```diff
@@ -2,13 +2,20 @@
 
 ## Prerequisites
 *  [Docker Desktop](https://www.docker.com/products/docker-desktop/) installed and running
-*  All funding-service-design apps (listed as `context` keys in [docker-compose.yml](docker-compose.yml) must be checked out in the parent directory on this repository
+*  **All** funding-service-design apps (listed as `context` keys in [docker-compose.yml](docker-compose.yml) must be cloned and checked out in the parent directory of this repository
+
+The necessary repositories to download are as follows:
+* https://github.com/communitiesuk/funding-service-design-post-award-data-store
+* https://github.com/communitiesuk/funding-service-design-account-store
+* https://github.com/communitiesuk/funding-service-design-authenticator
+* https://github.com/communitiesuk/funding-service-design-post-award-data-frontend
+* https://github.com/communitiesuk/funding-service-design-post-award-submit
```

Comment:
This info will bitrot if it gets out of sync from the `docker-compose.yml` file.

Maybe instead we can say the repositories can be found under communitiesuk github e.g. https://github.com/communitiesuk/funding-service-design-post-award-data-store ?

Reply:
Adding to this - perhaps it would at less risk of falling out of sync if we add this links as comments to the `docker-compose.yml` by each container